static/js/event/page.tsx

```diff
@@ -20,6 +20,43 @@
 
 import React from "react";
 
+import { Property } from "./types";
+
 export function Page(): JSX.Element {
```

Comment:
as mentioned in previous comment, we can remove this Page component

Reply:
Removed!